go.mod

```diff
@@ -220,6 +220,7 @@ require (
 	github.com/fatih/color v1.13.0 // indirect
 	github.com/felixge/httpsnoop v1.0.1 // indirect
 	github.com/fsnotify/fsnotify v1.5.4 // indirect
+	github.com/fvbommel/sortorder v1.0.1 // indirect
```

Comment:
I'm double checking but I don't think we actually need this anymore, it might have been something I needed beforehand when trying different k8s packages.

Reply:
`k8s.io/kubectl` needs which weirdly enough hasn't been an issue yet. This was something I added to fix a go.mod issues.